index.js

```diff
@@ -1,5 +1,74 @@
 'use strict';
 
+var colorConvert = require('color-convert');
+var supportsColor = require('supports-color');
+
+supportsColor = supportsColor || {};
+supportsColor.level = supportsColor.level || 0;
+
+colorConvert.rgb.rgb = function rgb (rgb) {
+	return rgb;
+};
+
+function getConverter (color) {
+	var has = function has (name) {
+		return color.hasOwnProperty(name);
+	}
+	switch (true) {
+		case has('r') && has('g') && has('b'): return 'rgb';
+		case has('h') && has('s') && has('l'): return 'hsl';
+		case has('h') && has('s') && has('v'): return 'hsv';
+		case has('h') && has('w') && has('b'): return 'hsv';
+		case has('c') && has('m') && has('y') && has('k'): return 'cmyk';
+		case has('css'): return 'keyword';
+		default: return null;
+	}
+}
+
+function getArrayFromKeyString(obj, str) {
+	if (str === 'keyword') {
+		return [obj['css']];
+	}
+
+	return str.split('').map(function(key) {
+		return obj[key];
+	});
+}
+
+var levels = [
+	function level0 () {
```

Comment:
It's for debugging. I usually always add function names for stack traces. @sindresorhus?